docs/TagSeqDiagram/ExecuteTagOp.puml

```diff
@@ -0,0 +1,34 @@
+@startuml
+'https://plantuml.com/sequence-diagram
+
+skinparam shadowing false
+participant ":TagCommand" as TagCommand
+
+mainframe **sd** Execute Tag Operation
+
+activate TagCommand
+
+alt TagOp == add
+TagCommand -> TagCommand:addTag(targetModule:Module)
+activate TagCommand
+TagCommand ->] :getTaskList()
+TagCommand <-] ::TaskList
+TagCommand ->] :addTag(tagDescription:String, taskIndex:int)
+TagCommand <-] ::Task
```

Comment:
I am not sure whether the Module Object can be omitted here according to the web.

Reply:
Alright, removed it